site/docs/master/install-overview.md

```diff
@@ -7,6 +7,38 @@ We strongly recommend that you use an [official release][1] of Velero. The tarba
 
 _The code and sample YAML files in the master branch of the Velero repository are under active development and are not guaranteed to be stable. Use them at your own risk!_
 
+## Install Velero
```

Comment:
Ah, I had made some similar changes in https://github.com/vmware-tanzu/velero/pull/1964. We can merge this and just follow up on my changes (which are a bigger reorg of this page) separately, or you can remove this file from this PR - up to you.